website/source/docs/providers/aws/r/glacier_vault.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: "aws"
+page_title: "AWS: aws_glacier_vault"
+sidebar_current: "docs-aws-resource-glacier-vault"
+description: |-
+  Provides a Glacier Vault.
+---
+
+# aws\_glacier\_vault
+
+Provides a Glacier Vault Resource. You can refer to the [Glacier Developer Guide](http://docs.aws.amazon.com/amazonglacier/latest/dev/working-with-vaults.html) for a full explanation of the Glacier Vault functionality
+
+## Example Usage
+
+```
+resource "aws_glacier_vault" "my_archive" {
+    name = "MyArchive"
+    
+    notification {
+      sns_topic = "arn:aws:sns:us-west-2:432981146916:MyArchiveTopic"
```

Comment:
How about using `aws_sns_topic` reference and/or even creating that SNS topic by defining `resource "aws_sns_topic" ...` below? I think it could help users to understand the relationship.